src/animation/AnimationMixer.js

```diff
@@ -199,42 +315,139 @@ THREE.AnimationMixer.prototype = {
 
 	update: function( deltaTime ) {
 
-		var mixerDeltaTime = deltaTime * this.timeScale;
-		this.time += mixerDeltaTime;
+		var actions = this._actions,
+			mixerDeltaTime = deltaTime * this.timeScale;
 
-		for ( var i = 0; i < this.actions.length; i ++ ) {
+		var time = this.time += mixerDeltaTime;
+		var accuIndex = this.accuIndex ^= 1;
```

Comment:
I'm not understanding why it is necessary to have ```accuIndex```?  Can you explain what I am missing?